src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -3,8 +3,43 @@ export interface ApproveDataOptions {
     orderID: string;
 }
 
+export enum StyleButtonLabel {
```

Comment:
I think this would be more easily consumed externally if exposed as strings rather than enums. Not all of our clients use typescript, so passing integers might not have a good readability for a JS consumer.

e.g:
```js
{
  layout: 1,
  color: 2,
  shape: 3
}
```

Reply:
Enum needs only for `_validateStyleParams()`. Because I don't want to duplicate code like 
`layout?: vertical | horizontal` and in _validateStyleParams() `if layout !== 'vertical' ...`

Actually, `layout: 'paypal'` - valid, because
`layout?: keyof typeof StyleButtonLayout` = `layout?: vertical | horizontal`

Reply:
The enum is not really just needed internally for `validateStyleParams`
This is also exposed externally. The styles can be provided as part of the argument to initialise the payment method, it will get eventually exported as part of the documentation.
https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/paymentinitializeoptions.md

I think you can still use the enum, but define strings instead of numbers. An example: https://github.com/bigcommerce/checkout-sdk-js/blob/06c19c2b35d76aa084ee317758588f43a84ba056/src/payment/strategies/square/square-form.ts#L78

Reply:
Yes, I know about documentation. But we still need validate this data, because script of payPal doesn't work with wrong style. PayPal doesn't validate it (so weird) and with one mistake in style param you don't see smart buttons

I tried to do like in `DigitalWalletType`. But in this case I can't validate without duplicate code. Now I check data like this `StyleButtonLabel[style.label]`, because `style.label` is key. If I define strings instead of numbers key doesn't match value

Or do you mean create enum with the same key and value? Like this 
`export enum StyleButtonLabel {
    paypal = 'paypal',
    checkout = 'checkout',
    buynow = 'buynow',
    pay = 'pay',
    installment = 'installment',
}`

Reply:
yeah, you can create the enum using strings instead of numbers so it's easier to be consumed externally.

Reply:
Ok, I'll do it
Thank you!